apps/wappush/js/cp_screen_helper.js

```diff
@@ -455,7 +452,7 @@ var CpScreenHelper = (function() {
                                                      authInfo);
       } catch (e) {
         message = finishConfirmDialog.querySelector('strong');
-        message.textContent = _(e.message);
+        navigator.mozL10n.setAttributes(message, e.message);
```

Comment:
Great catch. If there's a chance that e.message will not have an assigned l10nId in resource file, we would prefer you to follow the guideline.

The reason is, that quite soon we will want to turn on runtime language fallbacking which will basically start loading a fallback language if the resource cannot be find in the current language.

In this case, if the resource l10nid does not exist in any language, we will have to loop through the whole language fallback chain (imagine pt-BR, pt, en-US) loading two more languages fully only to find that this entity is not present in any of them.

It also makes your code more predictable and debuggable.

So we discourage try-and-fail method of using the resources. If possible, please, follow the guideline. Thanks!